app/controllers/admin_controller.rb

```diff
@@ -96,11 +96,11 @@ def includes
     []
   end
 
-  def filter
+  def filter(rel)
```

Comment:
is there a specific reason why these two filter functions look so vastly different?

Reply:
See description on other instance